CRFModuleValidation/CRFTaskFactory.swift

```diff
@@ -35,43 +35,4 @@ import ResearchSuite
 
 open class CRFTaskFactory: RSDFactory {
     
-    public enum CRFStepType: String {
```

Comment:
Is this class definition now empty? Do we still need it?

Reply:
We will need it for the recorders so I left it stubbed in so I could leave the override in place.